### PR TITLE
Climatic World Type Compat

### PR DESCRIPTION
**Pull request by valoeghese**

### CODE REVIEW

**Comment by valoeghese** [comment]
Wow I approve of this pr

File: src/main/java/com/terraformersmc/terrestria/Terrestria.java [hunk 1]
```diff
@@ -25,17 +35,27 @@ public void onInitialize() {
...
+			ClimaticWorldTypeCompat.init(configCache);
+		}
 
 		BIOME_CONFIG_HANDLER.save();
 
 		Registry.register(Registry.ITEM, new Identifier(MOD_ID, "log_turner"), new LogTurnerItem(new Item.Settings().group(ITEM_GROUP)));
 		LocateAny.register();
 	}
+
+	private static boolean isModLoaded(String id) {
```
Comment (coderbot16) [inline src/main/java/com/terraformersmc/terrestria/Terrestria.java]: not sure if this is necessary

File: src/main/java/com/terraformersmc/terrestria/compat/BiomeEnabledConfigCache.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.terraformersmc.terrestria.compat;
+
+import java.util.HashMap;
+
+import com.google.common.collect.Maps;
+
+public class BiomeEnabledConfigCache {
```
Comment (coderbot16) [inline src/main/java/com/terraformersmc/terrestria/compat/BiomeEnabledConfigCache.java]: why not just use a `HashSet<String>`?

**Review by coderbot16 — CHANGES_REQUESTED** [review]
Overall looks alright, but there's some things that need to be fixed up.

**Comment by valoeghese** [comment]
Can I merge this now

**Pull request merged**